File: Python Word examples/paragraphs/updateParagraphFormat.py
import spirecloudword
from spirecloudword import ParagraphFormat, Color, Border, Borders
from spirecloudword.configuration import Configuration as WordConfiguration

appId = "your id"
appKey = "your key"

configuration = WordConfiguration(appId, appKey)
api = spirecloudword.api.paragraphs_api.ParagraphsApi(configuration)

file_name = "Template.docx"
node_path = "sections/0"
index = 0
paragraph_format = ParagraphFormat()
paragraph_format._horizontal_alignment = "Center"
paragraph_format._left_or_inner_indent = 20
paragraph_format._right_or_outside_indent = 10
paragraph_format._mirror_indents = True
paragraph_format._first_line_indent = 30
paragraph_format._background_color = Color(60, 179, 113)
paragraph_format._before_spacing = 40
paragraph_format._after_spacing = 40
paragraph_format._before_auto_spacing = True
paragraph_format._after_auto_spacing = True
paragraph_format._line_space = 50
paragraph_format._line_spacing_rule = "Multiple"
paragraph_format._word_wrap = False
border = Border()
border._border_type = "Dot"
paragraph_format._borders = Borders(border)
extra = {'folder': 'input', 'dest_file_path': 'output/UpdateParagraphFormat.docx'}
result = api.update_paragraph_format(file_name, node_path, index, paragraph_format, **extra)

File: PHP Word Examples/ParagraphsApiDemo/updateParagraphFormat.php
<?php

use \Spire\Cloud\Word\Sdk\Client\Configuration;
use Spire\Cloud\Word\Sdk\Client\Model\Border;
use Spire\Cloud\Word\Sdk\Client\Model\Borders;
use \Spire\Cloud\Word\Sdk\Client\Api\ParagraphsApi;
use \Spire\Cloud\Word\Sdk\Client\Model\Color;
use \Spire\Cloud\Word\Sdk\Client\Model\ParagraphFormat;

$appId="your id";
$appKey="your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new ParagraphsApi($configuration);

$filename="UpdateParagraphFormat_1_Temp2.docx";
$folder="input";
$node_path = 'Section/0/Body/0';
$index = 0;
$dest_file_path = "Paragraphs/UpdateParagraphFormat_1.docx";
$format = new ParagraphFormat(array(
    'horizontal_alignment'=>ParagraphFormat::HORIZONTAL_ALIGNMENT_DISTRIBUTE,
    'left_or_inner_indent' => 20,
    'right_or_outside_indent' =>10,
    'mirror_indents' =>false,
    'first_line_indent' => 30,
    'background_color' => new Color(array('red' => 100, 'green' => 100, 'blue' => 100)),
    'before_spacing'=>40,
    'after_spacing'=>40,
    'before_auto_spacing'=>false,
    'after_auto_spacing'=>false,
    'line_space'=>50,
    'line_spacing_rule'=>ParagraphFormat::LINE_SPACING_RULE_EXACTLY,
    'word_wrap'=>true,
    'borders'=>new Borders(array(
        'top_border'=>new Border(array(
            'border_type'=>Border::BORDER_TYPE_DOUBLE
        ))
    ))
));
$storage = null;
$password = null;
$apiInstance->updateParagraphFormat($filename, $node_path, $index, $format, $dest_file_path, $folder, $storage, $password);